packages/cohort_banking/src/app.rs

```diff
@@ -82,23 +80,24 @@ impl Handler<TransferRequest> for BankingApp {
         let single_query_strategy = true;
         let state_provider = StateProviderImpl {
             database: Arc::clone(&self.database),
-            request: request_copy.clone(),
             single_query_strategy,
         };
+        let request_payload_callback = || state_provider.get_certification_candidate(certification_request.clone());
 
         let oo_inst = OutOfOrderInstallerImpl {
             database: Arc::clone(&self.database),
-            request: request_copy,
             detailed_logging: false,
             counter_oo_no_data_found: Arc::clone(&self.counter_oo_no_data_found),
             single_query_strategy,
         };
 
+        let oo_installer_callback = |payload: OOOInstallerPayload| oo_inst.install(payload);
```

Comment:
can it be a reference to function? `oo_inst::install`

Reply:
can I ask why you want to do that in the above example?

Reply:
It's a question, GK. Not suggesting to change anything. But if we can use reference maybe it is shorter that way?